test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -59,9 +62,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="appsettings.json">
```

Comment:
Was there a reason for this change?

Reply:
That file doesn't exist today: https://github.com/Azure/azure-functions-host/tree/dev/test/WebJobs.Script.Tests.Integration. I added it, then removed it, and I guess it cleaned itself up. I can undo this, though.